x-pack/plugins/observability_solution/apm/server/routes/entities/services/calculate_avg_metrics.ts

```diff
@@ -0,0 +1,31 @@
+import { mapValues } from 'lodash';
```

Comment:
Is this a duplicate from `observability_solution/apm/server/routes/entities/utils/calculate_avg_metrics.ts`?

Reply:
Yes, nice catch. forgot to delete it once removed to another folder.